app/controllers/categories_controller.rb

```diff
@@ -47,4 +48,12 @@ def category_params
   def category_find
     @category = Category.find_by(id: params[:id])
   end
+
+  def for_admin
```

Comment:
Move this method to Application controller. Or move controller under the Admin namespace.

Reply:
Updated